### DISCUSSION TITLE
Qutebrowser userscript repository?

### DISCUSSION THREAD

**Discussion by towalker** [qutebrowser/qutebrowser#5776] [General] [answered]
Hey guys,
Is there a a specific place where users can share their userscripts, something like  a qutebrowser userscript repository? This would be fantastic (even if extra care would have to be taken by the users to avoid malicious code).

**Comment by The-Compiler** [comment, marked as answer]
https://github.com/qutebrowser/qutebrowser/tree/master/misc/userscripts

**Reply by towalker** [reply]
Interesting, thanks. 
A follow-up question: for me to submit a new userscript to the list, should I submit it as a pull request (as recommended for modifications in general), or do userscripts additions should follow any other procedure?

**Reply by The-Compiler** [reply]
Yep, a PR will work just fine!
